tests/base.py

```diff
@@ -238,17 +239,17 @@ def tearDownClass(cls):
     httpretty.DELETE,
 ]
 def kill(*args, **kwargs):
-    raise httpretty.errors.UnmockedError
+    print('httppretty.kill', args, kwargs)
```

Comment:
switch over to logger call, w/o we get nothing from where the error occurred during tests